modules/bootstrapped/test/src/smithy4s/builder/EndpointBuilderSpec.scala

```diff
@@ -0,0 +1,58 @@
+package smithy4s
+
+import munit._
+
+class EndpointBuilderSpec extends FunSuite {
+
+  val endpoint = smithy4s.example.WeatherGen.endpoints.apply(2)
```

Comment:
**suggestion:** you can access this particular endpoint with `smithy4s.example.WeatherOperation.GetForecast` rather tahn indexing